src/openrct2/config/Config.cpp

```diff
@@ -249,7 +249,7 @@ namespace OpenRCT2::Config
             model->InvisibleTrees = reader->GetBoolean("invisible_trees", false);
             model->InvisibleScenery = reader->GetBoolean("invisible_scenery", false);
             model->InvisiblePaths = reader->GetBoolean("invisible_paths", false);
-            model->InvisibleSupports = reader->GetBoolean("invisible_supports", false);
+            model->InvisibleSupports = reader->GetBoolean("invisible_supports", true);
```

Comment:
I'm just confused on reverting this one, since it seems to have slipped in by accident 2y ago (see Hielke's comment)
https://github.com/OpenRCT2/OpenRCT2/commit/9e1b24d977efc5a436d331496d383bfdb55c96cf

Reply:
It was correct. These incredibly confusingly named options control what happens when you press the hotkey to hide them - `false` meaning see-through and `true` meaning completely invisible. Vanilla RCT made most things see-through when you hid them, but not supports. Hence why this differs from the rest, and why Hielke originally made that change.

The naming is very misleading, and so is the data type. In a next PR, I will fix this.